zerver/worker/queue_processors.py

```diff
@@ -1112,9 +1112,12 @@ def failure_processor(event: Dict[str, Any]) -> None:
             # Send a direct message notification letting the user who
             # triggered the export know the export finished.
             with override_language(user_profile.default_language):
-                content = _(
-                    "Your data export is complete and has been uploaded here:\n\n{public_url}"
-                ).format(public_url=public_url)
+                export_settings_link = (
+                    "[View and download exports](#organization/data-exports-admin)"
```

Comment:
This breaks internationalization. Read https://zulip.readthedocs.io/en/latest/translating/internationalization.html to see why this approach doesn't work.